server/doc.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+mongoose.promise = Promise;
+const Schema = mongoose.Schema;
+const date = require('date-and-time');
+
+const DocSchema = new Schema({
+  title: {type: String, required:true},
+  content: {type: String, required:true},
+  owner_id: {type: Schema.Types.ObjectId, ref: 'User'},
+  locked: Boolean,
+  revisions:[
+    {type: Schema.Types.ObjectId, ref: 'Revision'}
+  ],
+  date_created: {type: Date, 
```

Comment:
type should be moved to the next line for proper formatting, or just move default up

Reply:
Address this

Reply:
moved up